src/client/app/App.jsx

```diff
@@ -1,8 +1,10 @@
 'use strict';
```

Comment:
We do not want this file changed here. Please replace this file with the latest master and recommit the that state.